ui/values/dimensions.go

```diff
@@ -4,6 +4,7 @@ import "gioui.org/unit"
 
 var (
 	Size0_5 = unit.Dp(0.5)
+	Size28  = unit.Dp(28)
```

Comment:
Why don't we have this as the regular `MarginPadding28` and Size0_5 as `MarginPadding0p5`

Reply:
I'm slowly migrating to using `Size` as the prefix instead of `MargingPadding` as the values are not only used for margins and paddings.